build.sbt

```diff
@@ -126,10 +126,7 @@ lazy val js = project.in(file("js"))
   .settings(sharedSettings: _*)
   .settings(
     scalaJSStage in Global := FastOptStage,
-    libraryDependencies += "org.scala-js" %% "scalajs-test-interface" % scalaJSVersion,
-    // because Scala.js deprecated TestUtils but we haven't worked around that yet,
-    // see https://github.com/rickynils/scalacheck/pull/435#issuecomment-430405390
-    scalacOptions ~= (_ filterNot (_ == "-Xfatal-warnings"))
```

Comment:
If there are no deprecation warnings or warnings of any type in Platform.scala, then sure, `-Xfatal-warnings` can be re-enabled.

Reply:
Yup, there are no warnings of any kind anymore, thanks to this change. :)